postcss.config.js

```diff
@@ -1,7 +1,20 @@
+const autoprefixer = require('autoprefixer');
+const assets = require('postcss-assets');
+
+const CF_DIST = process.env.CLOUDFRONT_DIST;
```

Comment:
If we can't use environment variables at build time then we can't use postcss to solve the problem, so I'm not sure we should merge this at all. Does postcss have a way to update configuration at runtime?